### ISSUE TITLE
Only generates yaml for one group address

### ISSUE THREAD

**Issue by giggo** [JensVanhooydonck/xknx-configuration-importer#3]
I imported my group address report of about 300 group addresses but only got one group address in the yaml export. I can't see a log file indicating what went wrong. How can I track this down further?

**Comment by JensVanhooydonck** [comment]
Hi Giggo,

I still need to write the documentation. But your project needs to be configured with ETS Functions. These functions are then mapped to xKnx Functions. 

Is your project configured with functions? [https://support.knx.org/hc/en-us/articles/214497725-Creating-functions-and-linking-with-devices](url)

Jens

**Comment by giggo** [comment]
Bummer ;-) This doesn't seem to work with ETS. Just ETS Inside. 
I am on ETS5 at the moment. Too much hazzle to switch.

**Comment by JensVanhooydonck** [comment]
Actually it does work with ETS5. I just didn't find the documentation for it.
You can add Functions in Rooms inside ETS5.

**Comment by henne49** [comment]
yes I searched also ages, but you can use functions also in ETS5

click on a room, right click add -> functions -> here you get:
custom
switchable light
dimmable light
sun protection
heating (switching variable)
heating (continious variable)

**Comment by auredor** [comment]
Any idea if "Functions" is available in ETS4 ?

Thank you

**Comment by JensVanhooydonck** [comment]
I've just checked, and didn't find it there. Only in ETS5 i guess.

**Comment by auredor** [comment]
It's sad.

Thank you.

**Comment by PhilippeRaven** [comment]
Hi,
I'm having exactly the same problem, but I've got everything configured with functions in ETS 5. So currently I don't understand why I've got only 1 GA generated in the yaml file.

If there's a way to help me, thanks :)

**Comment by JensVanhooydonck** [comment]
> Hi,
> I'm having exactly the same problem, but I've got everything configured with functions in ETS 5. So currently I don't understand why I've got only 1 GA generated in the yaml file.
> 
> If there's a way to help me, thanks :)

Hi, 
Would it be possible to send me your knxproj-file?

**Comment by PhilippeRaven** [comment]
Since I posted, I put all my GA manually in my Home Assistant setup.
But for debug purpose, yes I can send it to you directly. Tell me by which way plase.
Thanks !

**Comment by JensVanhooydonck** [comment]
> Since I posted, I put all my GA manually in my Home Assistant setup.
> But for debug purpose, yes I can send it to you directly. Tell me by which way plase.
> Thanks !

Can you send it to jens@vanhooydonck.eu ?
